packages/rspack/src/lib/CacheFacade.js

```diff
@@ -59,6 +59,7 @@ class MultiItemCache {
 	getPromise() {
 		// @ts-expect-error
 		const next = i => {
+			// @ts-ignore if your typescript version >= 5.5, this line will throw an error
```

Comment:
Can we fix this error in >= 5.5

Reply:
this file needs to be translated to typescript, but not in this pull request